vite.config.ts

```diff
@@ -0,0 +1,22 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+
+export default defineConfig({
+  build: {
+    lib: {
+      // Could also be a dictionary or array of multiple entry points
```

Comment:
Is this from a tutorial or a real valuable comment?

Reply:
yup